pages/_app.jsx

```diff
@@ -144,28 +146,24 @@ export default function App(props) {
       <MatomoProvider value={matomoInstance}>
         <StylesProvider jss={jss}>
           <ThemeProvider theme={theme}>
-            <SessionProvider
-              sessionId="pod-browser"
-              restorePreviousSession
-              onSessionRestore={async (url) => {
```

Comment:
This is handled in `SessionReady` now, along with handlers for logout and errors.